### PR TITLE
Removed Default Codelabs Google Analytics ID

### PR DESCRIPTION
**Pull request by blainehussey**
This PR contains fixes to close out issue #67, which was opened to remove older (no longer used) Google Analytics and/or GTM containers. In this PR, we removed the default GA container `UA-123`, which appears to be the default one provided by Codelabs.

### CODE REVIEW

**Comment by blainehussey** [comment]
@TamimiGitHub - Here's a screenshot of the deployment on Staging which shows the original one has been removed now;

Screen Shot 2022-05-11 at 5 37 17 PM

File: codelab-elements/google-codelab-analytics/google_codelab_analytics_test.js [hunk 1]
```diff
@@ -103,51 +101,47 @@ testSuite({
...
-    window['ga']['getAll'] = mockGetAll;
-    window['ga']['create'] = mockCreate;
+    window["ga"]["getAll"] = mockGetAll;
+    window["ga"]["create"] = mockCreate;
 
     mockControl.$replayAll();
 
     document.body.appendChild(analytics);
-    analytics.setAttribute('codelab-gaid', 'UA-456');
+    analytics.setAttribute("codelab-gaid", "UA-456");
```
Comment (Mrc0113) [inline codelab-elements/google-codelab-analytics/google_codelab_analytics_test.js]: does this one need to be removed as well?

Comment (TamimiGitHub) [reply codelab-elements/google-codelab-analytics/google_codelab_analytics_test.js]: Yep, we will have to remove all occurrences of `analytics.setAttribute` in the test file if we want them to be removed from prod

Comment (blainehussey) [reply codelab-elements/google-codelab-analytics/google_codelab_analytics_test.js]: Good catch @Mrc0113, thanks for finding this.

Comment (blainehussey) [reply codelab-elements/google-codelab-analytics/google_codelab_analytics_test.js]: I've gone ahead and removed the one you found too @Mrc0113, it's been pushed, thanks again.

**Review by TamimiGitHub — APPROVED** [review]
LGTM! I also checked the preview link and confirmed that only the production tags are there. You can merge

**Pull request merged**